components/authorization/cluster/kustomization.yaml

```diff
@@ -1,5 +1,6 @@
 resources:
 - everyone-view.yaml
+- skabashnyuk-spi-vault-admin.yaml
```

Comment:
better to not use your nickname there, so later it can be extended by team members